routes/master/workers.js

```diff
@@ -17,7 +17,7 @@ config.workers.forEach(function (worker, index) {
 	if (worker.host.slice(-1) === '/') {
 		worker.host = worker.host.slice(0, -1);
 	}
-	if (!worker.host.includes('://')) {
+	if (worker.host.indexOf('://') === -1) {
```

Comment:
there is another checking with `includes`
https://github.com/yahoo/guerilla/blob/37d762fd3cb3a8593dd8b98221d17a3b5b83c0e1/lib/config.js#L72